config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-periodic-dra.yaml

```
@@ -235,3 +235,80 @@ periodics:
     annotations:
       testgrid-dashboards: sig-scalability-dra
       testgrid-tab-name: azure-dra-with-workload-master-scalability-100
+
+  - name: ci-kubernetes-e2e-gce-100-node-dra-with-workload
+    cluster: k8s-infra-prow-build
+    tags:
+      - "perfDashPrefix: gce-dra-100Nodes-with-workload"
+      - "perfDashBuildsCount: 270"
+      - "perfDashJobType: performance"
+    interval: 4h
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-e2e-scalability-common: "true"
+      preset-e2e-scalability-periodics: "true"
+      preset-e2e-scalability-periodics-master: "true"
+    annotations:
+      testgrid-dashboards: sig-scalability-dra
+      testgrid-tab-name: gce-dra-with-workload-master-scalability-100
+    decorate: true
+    decoration_config:
+      timeout: 3h
+    extra_refs:
+      - org: kubernetes
+        repo: kubernetes
+        base_ref: master
+        path_alias: k8s.io/kubernetes
+      - org: kubernetes
+        repo: perf-tests
+        base_ref: master
+        path_alias: k8s.io/perf-tests
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250710-e96fecb3d6-master
+          command:
+            - runner.sh
+            - /workspace/scenarios/kubernetes_e2e.py
+          args:
+            - --check-leaked-resources
+            - --cluster=
+            - --extract=ci/latest
+            - --gcp-node-image=gci
+            - --gcp-nodes=102
+            - --provider=gce
+            - -env=KUBE_FEATURE_GATES=DynamicResourceAllocation=true
```

@BenTheElder @johnbelamaric can I get some eyes on this? is this the correct way to enable the feature flag?

---

this looks _almost) correct to me. except that it should be:
```suggestion
            - --env=KUBE_FEATURE_GATES=DynamicResourceAllocation=true
```

(`env` is a long flag and should have two hyphens)

---

updated.